notebooks/apify_haystack_rag.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>Where can users find the parameters for <code>run_input</code>? Also, what does <code>actor_id</code> represent? What are the other alternatives? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I added this explanation into the notebook:

```
The `actor_id` and detailed description of input parameters (variable `run_input`) can be found on the [Website Content Crawler input page](https://apify.com/apify/website-content-crawler/input-schema).
```